src/components/Dashboard/TodoList/ConversionDetails.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div id="conversion-details" class="body-2">
+    <p class="mb-2">
+      BC Registries is missing information about this business.
+    </p>
+    <ul v-if="warnings.length > 0">
+      <li v-for="(item, index) in warnings" :key="index" class="pl-2">
+        {{item}}
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { TodoItemIF } from '@/interfaces'
+
+@Component({})
+export default class ConversionDetails extends Vue {
+  /** The subject filing. */
+  @Prop({ required: true }) readonly filing!: TodoItemIF
+
+  get warnings (): Array<string> {
+    return this.filing?.warnings || []
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/assets/styles/theme.scss";
+
+p, li {
+  color: $gray7;
+}
+</style>
```

Comment:
Looks like this:

![image](https://user-images.githubusercontent.com/10002889/195747076-5468e5ba-47ee-4beb-8785-814e0df62a33.png)